### DISCUSSION TITLE
Audio with no speech returns garbage transcript

### DISCUSSION THREAD

**Discussion by wacioc** [deepgram/community#524] [General help]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

Using an audio file with no speech to transcribe returns random pieces of transcript.
The attached audio file for example returned: "Bon Appetit!", but the sky is the limit. Sometimes it returns stuff like: "If you have any questions or other problems, please post them in the comments.".
This only happens with pre-recorded, but since our fallback mechanism uses pre-recorded when the real-time speech-to-text returns empty (which is correct in this scenario since there was no speech in the audio) it gives us an undefined behaviour.

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

POST /v1/listen?model=nova-2&smart_format=true&language=en-US

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

c1869e15-d785-41c9-b62c-ad6d62db7252

### If possible, please attach your code or paste it into the text box.

  const { result } = await deepgram.listen.prerecorded.transcribeFile(
    createReadStream(filePath),
    { model: "nova-2", smart_format: true, language: "en-US" }
  );

  const transcript = result?.results.channels[0]?.alternatives[0]?.transcript;

### If possible, please attach an example audio file to reproduce the issue.

https://github.com/deepgram/community/assets/15278138/ea81bc89-0e87-41cc-9f5e-99e319465184

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @wacioc, thanks for writing in with this question, and I'm sorry that this behavior is negatively affecting your workflow. It is a known issue that Nova-2 in particular can make some false predictions when given silent audio. We're actively working on improvements to the model to mitigate this. In the meantime, one option is to use our Nova model (`model=nova`). 

To understand your use case better - you're using streaming transcription, but if nothing is predicted, then you've recorded the audio and submit it to pre-recorded transcription at the end? If you're only seeing extra words transcribed with streaming the recorded audio, I wonder if it works sufficiently for you to trust that the blank streaming transcript is indeed non-speech audio. Or have you observed many other cases where our streaming transcript is empty but the audio actually does contain speech? Streaming and pre-recorded transcription may differ slightly, but will generally be very similar.

**Reply by wacioc** [reply]
Thanks for your answer. We'll switch back to "nova" until' there's a fix for this (if you could also write in here when that happens it would be highly appreciated). We want to keep the fallback mechanism since there's a lot that can go wrong when sockets are involved :D.

**Reply by wacioc** [reply]
Actually, just tried using "nova" and the same audio with no speech in it returned and I quote: "美味しいです。 見てくれてありがとう。"

**Reply by jkroll-deepgram** [reply]
If possible, can you please share a request ID for each of the Nova-2 and Nova requests? That will help me look into it further.

**Reply by wacioc** [reply]
c1869e15-d785-41c9-b62c-ad6d62db7252 here is the request id (which I had already shared above)  for the nova-2 request.

**Reply by wacioc** [reply]
Any updates on this?

**Reply by jkroll-deepgram** [reply]
Hi @wacioc, to confirm, you're submitting a recorded audio file, with the parameters `model=nova-2&smart_format=true&language=en-US` (or `model=nova` instead of `model=nova-2` given my suggestion)?

Here's a curl request I'm running with the audio file you originally attached to the issue. I see that your audio file is 5.5 seconds of silence.
```
curl \
-X POST \
-H "Authorization: Token <token>" \
--data-binary @295747269-ea81bc89-0e87-41cc-9f5e-99e319465184.mp4 \
"https://api.deepgram.com/v1/listen?model=nova-2&smart_format=true&language=en-US"
```

The transcript is:
```
{
  "metadata": {
    "transaction_key": "deprecated",
    "request_id": "8f07fb97-446f-4513-ad36-29db14574e30",
    "sha256": "1b25325918bd835d8af2c65ebabd042ed06e86a147d5808b9010a32e93974994",
    "created": "2024-01-16T21:50:29.851Z",
    "duration": 5.44,
    "channels": 1,
    "models": [
      "a375937a-9156-40cb-940d-6a1103040ef1"
    ],
    "model_info": {
      "a375937a-9156-40cb-940d-6a1103040ef1": {
        "name": "2-general-nova",
        "version": "2023-11-14.3290",
        "arch": "nova-2"
      }
    }
  },
  "results": {
    "channels": [
      {
        "alternatives": [
          {
            "transcript": "",
            "confidence": 0,
            "words": []
          }
        ]
      }
    ]
  }
}
```

When I re-run the same request but with Nova, I get a nearly identical response:

```
curl \
-X POST \
-H "Authorization: Token <token>" \
--data-binary @295747269-ea81bc89-0e87-41cc-9f5e-99e319465184.mp4 \
"https://api.deepgram.com/v1/listen?model=nova&smart_format=true&language=en-US"
```

```
{
  "metadata": {
    "transaction_key": "deprecated",
    "request_id": "0c34c51f-3c29-4a2f-a3a9-ab78e8ac5086",
    "sha256": "1b25325918bd835d8af2c65ebabd042ed06e86a147d5808b9010a32e93974994",
    "created": "2024-01-16T21:51:10.338Z",
    "duration": 5.44,
    "channels": 1,
    "models": [
      "b227621c-0920-4128-b4b5-a3e0f525d2d7"
    ],
    "model_info": {
      "b227621c-0920-4128-b4b5-a3e0f525d2d7": {
        "name": "general-nova",
        "version": "2023-07-28.18608",
        "arch": "nova"
      }
    }
  },
  "results": {
    "channels": [
      {
        "alternatives": [
          {
            "transcript": "",
            "confidence": 0,
            "words": []
          }
        ]
      }
    ]
  }
}
```

I see a similarly blank transcript when trying to reproduce given your request ID `c1869e15-d785-41c9-b62c-ad6d62db7252`.

Are you able to share a transcript response from one of these cases, or even better a full code sample or curl command demonstrating the word output? While I find your observations plausible as our models are not optimized for the use case of transcribing minutes of silence, I'm blocked on troubleshooting until I have an instance where I'm able to reproduce and examine the behavior.
